packages/@glimmer/validator/index.ts

```diff
@@ -1,3 +1,17 @@
+import { symbolFor, getGlobal } from './lib/utils';
+
+const globalObj = getGlobal();
+
+const GLIMMER_VALIDATOR_REGISTRATION = symbolFor('GLIMMER_VALIDATOR_REGISTRATION');
+
+if ((globalObj as any)[GLIMMER_VALIDATOR_REGISTRATION] === true) {
+  throw new Error(
+    'Two versions of the `@glimmer/validator` library have been included in this application. `@glimmer/validator` depends on having a single version of the package in use at any time in an application. You must dedupe your build to remove the duplicate packages in order to prevent this error.'
```

Comment:
> Two versions of the `@glimmer/validator` library have been included in this application.

FWIW, this isn't quite correct. 🤔 You could end up with this error if you have the same version in your output twice (which I wouldn't consider "two versions"). I don't really know how to tweak the prose to make that particular quibble moot though.

Reply:
Updated, I think it's a bit better now. Thanks for pointing that out